beacon-chain/db/kv/powchain.go

```diff
@@ -0,0 +1 @@
+package kv
```

Comment:
What's up with this?

Reply:
+1 on this package name, it can be specified in the .proto file

Reply:
removed it